src/ecosystem/connectivity.jl

```diff
@@ -26,7 +26,7 @@ NOTE: Transposes transitional probability matrix if `swap == true`
 
 # Returns
 NamedTuple:
-- `TP_data` : Matrix, containing the transition probability for all sites
+- `conn` : Matrix, containing the connectivity for all sites
```

Comment:
Maybe add a note in brackets `(this accounts for larvae which do not settle and so rows may not sum to 1)` or something like that